biome.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.7.3/schema.json",
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": false,
+    "indentStyle": "space",
+    "indentWidth": 2,
+    "lineEnding": "lf",
+    "lineWidth": 100,
+    "attributePosition": "auto"
+  },
+  "organizeImports": {
+    "enabled": true
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "style": {
+        "noParameterAssign": "off",
+        "noDefaultExport": "warn",
+        "useCollapsedElseIf": "error"
+      },
+      "suspicious": {
+        "noExplicitAny": "off",
```

Comment:
`unknown` is better than `any`. If you _really_ must use `any`, it may be preferable to disable the rule at line level with an explanation, as `any`s have a way of sneaking up on you. A deep `any` may sweep type errors under the rug high up the dependency chain, without any signs to the developer, until you hover over the variable and it turns out to be an `any`

Reply:
In the main branch we currently only use TypeScript in our `index.d.ts` definition files and I would like to keep it as unmodified as possible in this PR. Thus I will turn this into `warn` as well. With the TypeScript migration in #683, the entire types will be overhauled anyways. Thanks for bringing this up!